saleor/product/utils/variants.py

```diff
@@ -20,7 +22,9 @@ def generate_and_set_variant_name(variant: "ProductVariant", sku: str):
         attribute_rel
     ) in variant_selection_attributes.iterator():  # type: AssignedVariantAttribute
         values_qs = attribute_rel.values.all()
-        translated_values = [str(value.translated) for value in values_qs]
+        translated_values = [
+            truncatechars(str(value.translated), 25) for value in values_qs
```

Comment:
Why we trim this value here, where is it causing problems now?

Reply:
@maarcingebala 
![image](https://user-images.githubusercontent.com/5078402/122019879-88f56600-cdc4-11eb-9d49-3c81aa18137d.png)

Reply:
Looks like a dashboard issue ;) I think it's the frontend's responsibility to handle rendering long values.

Reply:
@maarcingebala remember variant name has `max_length=255` on the model 🤔

Reply:
Still, there are German words longer than 25 characters. If needed, we can truncate the resulting string instead of trying to trim each independent part.

Reply:
Ok make sense. Let's handle it on to the frontend 🙆‍♂️